python/py-stanfordnlp/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python  1.0
+PortGroup           java    1.0
+PortGroup           github  1.0
+
+github.setup        stanfordnlp stanfordnlp 0.2.0 v
+name                py-stanfordnlp
+revision            0
+
+categories          python textproc java
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         A Python NLP Library for Many Human Languages
+long_description    The Stanford NLP Group's official Python NLP \
+                    library. It contains packages for running our \
+                    latest fully neural pipeline from the CoNLL 2018 \
+                    Shared Task and for accessing the Java Stanford \
+                    CoreNLP server.
+
+homepage            https://stanfordnlp.github.io/stanfordnlp/
+
+github.livecheck.regex  {([0-9.]+)}
+
+checksums           rmd160  95e41e706e48120a6c3923c987404538b3da37af \
+                    sha256  549c6993d0088583c35eab33f8c28e5dfdfe7c5153370d4c01bc26de56cdc46e \
+                    size    130852
+
+# Required java version
+java.version        11+
+# JDK port to install if required java not found
+java.fallback       openjdk13
+
+# Support python versions
+python.versions        27 37
```

Comment:
One more addition here. Can python 3.8 be added? We are adding that to all of the python sub-ports now that python 3.8 has been released. Also, please column align it like the lines above.

Reply:
Thanks—done.